Dockerfile

```diff
@@ -1,12 +1,9 @@
 FROM sameersbn/ubuntu:14.04.20170503
 MAINTAINER sameer@damagehead.com
 
-ENV GITLAB_VERSION=9.1.4 \
+ENV GITLAB_VERSION=9.2.0 \
     RUBY_VERSION=2.3 \
     GOLANG_VERSION=1.6.3 \
-    GITLAB_SHELL_VERSION=5.0.2 \
```

Comment:
At the moment I don't want to have them dynamically in the `install.sh`. So please re add them there because it's easier to change the version and have an overview which version is currently used.